### PR TITLE
Show equation being plotted

### PR DESCRIPTION
**Pull request by jeanimal**
Right below the degree text box,  added a new textbox that is disabled (uneditedable) with the equation actually being plotted, based on the degree.  For example
degree  equation
1             a0 + x^1 = 0
2            a0 + a1 x^1 + x^2 = 0
3            a0 + a1 x^1 + a2 x^2 + x^3 = 0

I had the idea to add the equation because I was confused.  The coefficient labels on the plot did not obviously match the equation at the top of the "Explanation" section-- e.g. for degree n there is no a_n-- so I thought there was a mistake.

Screen Shot 2019-05-23 at 8 29 06 PM

### CODE REVIEW

**Comment by duetosymmetry** [comment]
Please take a look at [this page on the MathJax docs](https://docs.mathjax.org/en/latest/advanced/typeset.html) to see how to programmatically ask MathJax to typeset an equation after load time. I suspect it's not possible to have typeset math inside of a textarea? In that case, I'm not sure what's the right UI approach to make it clear that the equation is an output that's linked to the degree input.

**Comment by jeanimal** [comment]
New pull request for Latex version.
https://github.com/duetosymmetry/web-site/pull/3
